src/phantom.js

```diff
@@ -168,7 +168,7 @@ export default class Phantom {
             } else {
                 r = typeof val === 'function' ? val.toString() : val;
 
-                if(typeof val === 'function' && r.includes('=>')) {
+                if(typeof val === 'function' && r.indexOf('(') === 0) {
```

Comment:
@elias-winberg this won't work. I had a hard time figuring out how to to search for `=>` because there are so many variations for `() => {}`. According to [mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) documentations, here are the rules...

> (param1, param2, …, paramN) => { statements }
> (param1, param2, …, paramN) => expression
>          // equivalent to:  => { return expression; }
> 
> // Parentheses are optional when there's only one parameter:
> (singleParam) => { statements }
> singleParam => { statements }
> 
> // A function with no parameters requires parentheses:
> () => { statements }

So `one => true` would actually be a valid syntax. Yours doesn't account for that.